src/Illuminate/Database/Concerns/ManagesTransactions.php

```diff
@@ -34,14 +34,10 @@ public function transaction(Closure $callback, $attempts = 1)
             // If we catch an exception we'll rollback this transaction and try again if we
             // are not out of attempts. If we are out of attempts we will just throw the
             // exception back out and let the developer handle an uncaught exceptions.
-            catch (Exception $e) {
+            catch (Throwable $e) {
```

Comment:
Is there a slight change of behavior on this one? Previously on Throwable we used to just rollBack and re-throw.